packages/web/src/components/spa/HackAUBG/MentorsSection/MentorsSection.jsx

```diff
@@ -40,7 +40,16 @@ export const MentorsSection = () => {
     useEffect(() => {
         getMentors();
     }, []);
-    return <>{renderMentors()}</>;
+    return (
+        <div className="coming-soon">
+            <>{renderMentors()}</>
+
+            <div className="coming-soon-text">
```

Comment:
This div should be rendered only if the mentors list is empty, put it in the renderMentors function